app/templates/events/trackVolunteers.html

```diff
@@ -44,59 +44,241 @@
+                          name="inputHours_{{participant.user.username}}"
+                          style="max-width: 90px;"
+                          type ="number"
+                          min="0"
+                          step="0.01"
+                            {% if participant.hoursEarned == None %}
+                              value=""
+                            {% else %}
+                              value="{{participant.hoursEarned}}"
+                            {% endif %}
+                        />
+                      </td>
+                      <td style="text-align:center">
+                        <input
+                          type="checkbox"
+                          class='attendanceCheck'
+                          name="checkbox_{{participant.user.username}}"
+                          {% if event.isPast %}
+                            checked
+                          {% endif %} />
+                      </td>
+                    {% else %} {# end hiding the hours and attended inputs #}
+                      <td style="text-align:center">
+                        {% if participant.user.username not in trainedParticipantsList %}
+                          <span class="bi bi-x-lg" aria-label="Did not complete all trainings."></span>
+                        {% else %}
+                          <span class="bi bi-check-lg" aria-label="Completed all trainings."></span>
+                        {% endif %}
+                      </td>
+                    {% endif %}
+                    <td style="text-align:center">
+                      <button
+                        type="button"
+                        class="removeVolunteer btn btn-danger btn-sm"
+                        id="{{participant.user.username}}"
+                        aria-label="remove volunteer">
+                        <span class="bi bi-trash h5 align-middle"></span>
+                      </button>
+                    </td>
+                  </tr>
+                {% endfor %}
               {% endif %}
-              <td>
-                <input
-                  class="form-control"
-                  type='hidden'
-                  name="username{{loop.index|string}}"
-                  id="{{participant.user.username}}"
-                  value="{{participant.user}}" />
-                  {{participant.user.firstName}} {{participant.user.lastName}}
-                  {% if participant.user in bannedUsers %}
+            </tbody>
+          </table>
+          {% if event.isPast %}
+            <div class="row">
+              <div class="col"></div>
+              <button type="submit" class="btn btn-primary float-end nowrap col-auto mx-3">Save</button>
+            </div>
+          {% endif %}
+        </div>
+      </div>
+    </div>
+</form>
+    {% if event.isRsvpRequired and event.rsvpLimit and not event.isPast and eventWaitlistData %}
+    <!-- Waitlist Table -->
```

Comment:
It's good to have this comment here, but let's make it even more of a visual break. It's nice to break up these different segments with more of a section header kind of comment.